frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -180,16 +195,62 @@ export default function OntologyDagView({
   // Animation duration - initially zero so the animation doesn't play on load
   const [duration, setDuration] = useState(0);
 
+  // construct explorer URL
+  const formattedSelectedOrganism = selectedOrganism
```

Comment:
This is logic for constructing explorer URL

Reply:
feels like a good opportunity to extract this out to be a standalone function in a `utils.ts` file and import and use it in this index file to keep this main file clean?